pex/resolver.py

```diff
@@ -418,7 +418,10 @@ def finalize_install(self, install_requests):
     runtime_key_dir = os.path.join(self.installation_root, wheel_dir_hash)
     with atomic_directory(runtime_key_dir) as work_dir:
       if work_dir:
-        os.symlink(self.install_chroot, os.path.join(work_dir, self.request.wheel_file))
+        source_path = os.path.join(work_dir, self.request.wheel_file)
```

Comment:
Maybe add a brief comment?

# NB: It's important to use relative paths here because ..

Reply:
Added.